### PR TITLE
Adding disabled test for Startup Singleton EJB with Data respository

### PR DESCRIPTION
**Pull request by mswatosh**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

### CODE REVIEW

**Comment by mswatosh** [comment]
#build ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=3e7040f9-1f26-41ee-b44c-694dd3bb5c49) - :white_check_mark: completed successfully!)

_Note: Target locations of links might be accessible only to IBM employees._

File: dev/io.openliberty.data.internal_fat_datastore/test-applications/DataStoreTestApp/src/test/jakarta/data/datastore/ejb/StartupSingletonEJB.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/**
+ *
+ */
```
Comment (njr-11) [inline dev/io.openliberty.data.internal_fat_datastore/test-applications/DataStoreTestApp/src/test/jakarta/data/datastore/ejb/StartupSingletonEJB.java:3]: /*******************************************************************************
 * Copyright (c) 2024 IBM Corporation and others.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Public License 2.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/legal/epl-2.0/
 *
 * SPDX-License-Identifier: EPL-2.0
 *
 * Contributors:
 *     IBM Corporation - initial API and implementation
 *******************************************************************************/

**Review by njr-11 — DISMISSED** [review]
Looks good, but needs a copyright.

File: dev/io.openliberty.data.internal_fat_datastore/test-applications/DataStoreTestApp/src/test/jakarta/data/datastore/ejb/StartupSingletonEJB.java (hunk 1 above)
Comment (njr-11) [inline dev/io.openliberty.data.internal_fat_datastore/test-applications/DataStoreTestApp/src/test/jakarta/data/datastore/ejb/StartupSingletonEJB.java]:
```suggestion
/*******************************************************************************
 * Copyright (c) 2024 IBM Corporation and others.
 * All rights reserved. This program and the accompanying materials
 * are made available under the terms of the Eclipse Public License 2.0
 * which accompanies this distribution, and is available at
 * http://www.eclipse.org/legal/epl-2.0/
 *
 * SPDX-License-Identifier: EPL-2.0
 *
 * Contributors:
 *     IBM Corporation - initial API and implementation
 *******************************************************************************/
```

**Comment by mswatosh** [comment]
#build ([view Open Liberty Personal Build](https://libh-proxy1.fyre.ibm.com/cognitive/pipelineAnalysis.html?pipelineId=cb62855b-7da5-4d28-b217-24f0aa36d75a) - :white_check_mark: completed successfully!)

_Note: Target locations of links might be accessible only to IBM employees._

**Review by njr-11 — APPROVED** [review]

**Pull request merged**